packages/next-swc/crates/next-core/js/src/overlay/internal/container/RuntimeError.tsx

```diff
@@ -152,98 +152,84 @@ export function RuntimeError({ error }: RuntimeErrorProps) {
             onClick={toggleAll}
             className="runtime-error-collapsed-action"
           >
-            {all ? "Hide" : "Show"} collapsed frames
+            {all ? 'Hide' : 'Show'} collapsed frames
           </button>
         </>
       ) : undefined}
     </>
-  );
+  )
 }
 
 function HotlinkedText(props: { text: string }) {
-  const { text } = props;
+  const { text } = props
 
-  const linkRegex = /https?:\/\/[^\s/$.?#].[^\s)'"]*/i;
+  const linkRegex = /https?:\/\/[^\s/$.?#].[^\s)'"]*/i
   return (
     <>
       {linkRegex.test(text)
-        ? text.split(" ").map((word, index, array) => {
+        ? text.split(' ').map((word, index, array) => {
             if (linkRegex.test(word)) {
-              const link = linkRegex.exec(word);
+              const link = linkRegex.exec(word)
               return (
                 <React.Fragment key={`link-${index}`}>
                   {link && (
                     <a href={link[0]} target="_blank" rel="noreferrer noopener">
                       {word}
                     </a>
                   )}
-                  {index === array.length - 1 ? "" : " "}
+                  {index === array.length - 1 ? '' : ' '}
                 </React.Fragment>
-              );
+              )
             }
             return index === array.length - 1 ? (
               <React.Fragment key={`text-${index}`}>{word}</React.Fragment>
             ) : (
               <React.Fragment key={`text-${index}`}>{word} </React.Fragment>
-            );
+            )
           })
         : text}
     </>
-  );
+  )
 }
 
 type RuntimeErrorsDialogBodyProps = {
-  isLoading: boolean;
-  readyErrors: ReadyRuntimeError[];
-  "data-hidden"?: boolean;
-};
+  items: ReadyRuntimeError[]
+  message: string
+  'data-hidden'?: boolean
+}
 
 export function RuntimeErrorsDialogBody({
-  isLoading,
-  readyErrors,
-  "data-hidden": hidden = false,
+  items: readyErrors,
+  message,
+  'data-hidden': hidden = false,
   className,
   ...rest
-}: RuntimeErrorsDialogBodyProps & Omit<DialogBodyProps, "children">) {
+}: RuntimeErrorsDialogBodyProps & Omit<DialogBodyProps, 'children'>) {
   const [activeError, { previous, next }, activeIdx] =
-    usePagination(readyErrors);
-
-  if (isLoading) {
-    return (
-      <DialogBody
-        {...rest}
-        data-hidden={hidden}
-        className={clsx("runtime-errors", className)}
-      >
-        <h1 id="nextjs__container_errors_label">Resolving Source Maps...</h1>
```

Comment:
if I'm understanding it correctly, we will now resolve the source maps and show the error overlay after?

Reply:
I think we should show the unresolved error instantly and resolve the source map in background.

Reply:
`readyErrors` only gets resolved entries, so without the loading case the error overlay will only show after the first error was resolved

Reply:
yeah, let's change that?

Reply:
changed that